syntax = "proto3";

service StorageService {
  rpc GetHistoricalData (DataRequest) returns (DataResponse) {}
}

message DataRequest {
  string sensor_id = 1;
}

message DataResponse {
  repeated string data = 1;
}